aspnetcore/fundamentals/configuration.md

```diff
@@ -77,7 +77,7 @@ Configuration considerations:
 * `IOptionsSnapshot` can reload configuration data when it changes. Use `IOptionsSnapshot` if you need to reload configuration data.  See [IOptionsSnapshot](#ioptionssnapshot) for more information.
 * Configuration keys are case insensitive.
 * A best practice is to specify environment variables last, so that the local environment can override anything set in deployed configuration files.
-* **Never** store passwords or other sensitive data in configuration provider code or in plain text configuration files. You also shouldn't use production secrets in your development or test environments. Instead, specify secrets outside the project tree, so they cannot be accidentally committed into your repository. Learn more about [Working with Multiple Environments](environments.md) and managing [Safe storage of app secrets during development](../security/app-secrets.md).
+* **Never** store passwords or other sensitive data in configuration provider code or in plain text configuration files. Do not use production secrets in your development or test environments. Instead, specify secrets outside the project tree, so they cannot be accidentally committed into your repository. Learn more about [Working with Multiple Environments](environments.md) and managing [safe storage of app secrets during development](../security/app-secrets.md).
```

Comment:
Do not --> Don't

if S should be lower case, W in Working with... should also be LC